[2023-03-23T02:30:12.206+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_minio_s3_v04.sensor_minio_s3 manual__2023-03-23T02:22:45.719276+00:00 [queued]>
[2023-03-23T02:30:12.227+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_minio_s3_v04.sensor_minio_s3 manual__2023-03-23T02:22:45.719276+00:00 [queued]>
[2023-03-23T02:30:12.227+0000] {taskinstance.py:1282} INFO - 
--------------------------------------------------------------------------------
[2023-03-23T02:30:12.228+0000] {taskinstance.py:1283} INFO - Starting attempt 4 of 4
[2023-03-23T02:30:12.229+0000] {taskinstance.py:1284} INFO - 
--------------------------------------------------------------------------------
[2023-03-23T02:30:12.261+0000] {taskinstance.py:1303} INFO - Executing <Task(S3KeySensor): sensor_minio_s3> on 2023-03-23 02:22:45.719276+00:00
[2023-03-23T02:30:12.267+0000] {standard_task_runner.py:55} INFO - Started process 1826 to run task
[2023-03-23T02:30:12.271+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dag_with_minio_s3_v04', 'sensor_minio_s3', 'manual__2023-03-23T02:22:45.719276+00:00', '--job-id', '229', '--raw', '--subdir', 'DAGS_FOLDER/minio_s3.py', '--cfg-path', '/tmp/tmpa4sf7z8d']
[2023-03-23T02:30:12.275+0000] {standard_task_runner.py:83} INFO - Job 229: Subtask sensor_minio_s3
[2023-03-23T02:30:12.384+0000] {task_command.py:388} INFO - Running <TaskInstance: dag_with_minio_s3_v04.sensor_minio_s3 manual__2023-03-23T02:22:45.719276+00:00 [running]> on host 9783a48d7fca
[2023-03-23T02:30:12.579+0000] {taskinstance.py:1511} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=ThinhHuynh
AIRFLOW_CTX_DAG_ID=dag_with_minio_s3_v04
AIRFLOW_CTX_TASK_ID=sensor_minio_s3
AIRFLOW_CTX_EXECUTION_DATE=2023-03-23T02:22:45.719276+00:00
AIRFLOW_CTX_TRY_NUMBER=4
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-23T02:22:45.719276+00:00
[2023-03-23T02:30:12.582+0000] {s3.py:99} INFO - Poking for key : s3://***/data.csv
[2023-03-23T02:30:12.599+0000] {base.py:73} INFO - Using connection ID 'minio_conn' for task execution.
[2023-03-23T02:30:12.601+0000] {connection_wrapper.py:337} INFO - AWS Connection (conn_id='minio_conn', conn_type='aws') credentials retrieved from login and password.
[2023-03-23T02:30:16.327+0000] {base.py:228} INFO - Success criteria met. Exiting.
[2023-03-23T02:30:16.365+0000] {taskinstance.py:1326} INFO - Marking task as SUCCESS. dag_id=dag_with_minio_s3_v04, task_id=sensor_minio_s3, execution_date=20230323T022245, start_date=20230323T023012, end_date=20230323T023016
[2023-03-23T02:30:16.423+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-03-23T02:30:16.464+0000] {taskinstance.py:2585} INFO - 0 downstream tasks scheduled from follow-on schedule check
